docs/how_to/multimodal_training_and_serving.md

```diff
@@ -135,6 +135,18 @@ class PairTextImage(BaseDoc):
     image: ImageDoc
 ```
 
+You then need to forward declare the following types, this will allow the objects to be properly pickled and unpickled.
```

Comment:
What does "forward declare" mean? Declare beforehand?

Reply:
`forward declaration` is a common term in programming to say that u make aware to the context the existence of a symbol, method, variable, etc... before it exists. Like for some compilers and so it is necessary, it is necessary here for the pickler to know that that class exist.

The thing is that `DocVec[specific_document]` is a dynamically created class, and here we are saying to the modules that such a class exists